priv/cabbage/apps/itest/test/features/watcher_info_api.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Watcher info
+
+  Scenario: Alice want to use watcher info api
```

Comment:
```suggestion
  Scenario: Alice wants to use retrieve her UTXO information after deposit
```

Reply:
Not updated yet I think, so un-resolving...